lib/js/components/Buttons/Button/Button.vue

```diff
@@ -147,6 +148,7 @@ export default {
 		this.SIZES = BUTTON_SIZES;
 		this.STATES = BUTTON_STATES;
 		this.ELEVATIONS = BUTTON_ELEVATIONS;
+		this.BUTTON_COLORS_WITH_ICON_COLORS = BUTTON_COLORS_WITH_ICON_COLORS;
```

Comment:
https://bethinkteam.slack.com/archives/C01N1F1QMUY/p1650525683804989?thread_ts=1650477770.866669&cid=C01N1F1QMUY

In `platform` code we already started migration to `data()`

Reply:
Oh nice, so I refactored it by the way in `Button` and `IconButton` in this PR.